src/P.hs

```diff
@@ -73,6 +74,7 @@ import           Data.Int as X
 import           Data.Ord as X
 import           Data.Tuple as X
 import           Data.Traversable as X
+import           Data.Text as X (Text)
```

Comment:
Yeah it will, re-exports aren't resolved wrt import conflicts I think. People will have to manually upgrade for things to break though.